--- m2crypto-0.11/M2Crypto/SMIME.py.wiget	2002-12-23 04:48:35.000000000 +0100
+++ m2crypto-0.11/M2Crypto/SMIME.py	2003-10-08 16:18:10.682936771 +0200
@@ -54,6 +54,9 @@
     def write_der(self, bio):
         return m2.pkcs7_write_bio_der(self.pkcs7, bio._ptr())
 
+    def get0_signers(self, certs, flags = 0):
+      return X509.X509_Stack(m2.pkcs7_get0_signers(self.pkcs7, certs.stack, flags), 1)
+
 
 def load_pkcs7(p7file):
     bio = m2.bio_new_file(p7file, 'r')
--- m2crypto-0.11/SWIG/_pkcs7.i.wiget	2003-10-08 16:24:43.072269817 +0200
+++ m2crypto-0.11/SWIG/_pkcs7.i	2003-10-08 16:25:00.601515310 +0200
@@ -170,5 +170,10 @@
 int smime_crlf_copy(BIO *in, BIO *out) {
     return SMIME_crlf_copy(in, out, PKCS7_TEXT);
 }
+
+/* return STACK_OF(X509)* */
+STACK *pkcs7_get0_signers(PKCS7 *p7, STACK *certs, int flags) {
+	return PKCS7_get0_signers(p7,certs,flags);
+}
 %}
 
