--- m2crypto-0.11/M2Crypto/SMIME.py.wiget	2002-12-23 04:48:35.000000000 +0100
+++ m2crypto-0.11/M2Crypto/SMIME.py	2003-09-16 13:36:00.000000000 +0200
@@ -4,7 +4,7 @@
 
 RCS_id='$Id$'
 
-import BIO, EVP, X509, Err
+import BIO, EVP, X509, Err, util
 import m2
 
 PKCS7_TEXT	= m2.PKCS7_TEXT
@@ -117,16 +117,16 @@
 m2.smime_init(SMIME_Error)
 
 class SMIME:
-    def load_key(self, keyfile, certfile=None):
+    def load_key(self, keyfile, certfile=None, callback=util.passphrase_callback):
         if certfile is None:
             certfile = keyfile
-        self.pkey = EVP.load_key(keyfile)
+        self.pkey = EVP.load_key(keyfile, callback)
         self.x509 = X509.load_cert(certfile)
 
-    def load_key_bio(self, keybio, certbio=None):
+    def load_key_bio(self, keybio, certbio=None, callback=util.passphrase_callback):
         if certbio is None:
             certbio = keybio
-        self.pkey = EVP.load_key_bio(keybio)
+        self.pkey = EVP.load_key_bio(keybio, callback)
         self.x509 = X509.load_cert_bio(certbio)
 
     def set_x509_stack(self, stack):
