diff -Nru m2crypto-0.18.1.orig/SWIG/_lib.h m2crypto-0.18.1/SWIG/_lib.h
--- m2crypto-0.18.1.orig/SWIG/_lib.h	2007-10-10 18:13:27.272883250 +0200
+++ m2crypto-0.18.1/SWIG/_lib.h	2007-10-10 18:11:49.078746500 +0200
@@ -1,7 +1,7 @@
 /* Copyright (c) 1999 Ng Pheng Siong. All rights reserved. */
 /* $Id$ */
 
-#if PY_VERSION_HEX < 0x02050000
+#if PY_VERSION_HEX < 0x02050000 && !defined(PY_SSIZE_T_MIN)
 typedef int Py_ssize_t;
 #endif
 
